lib/awful/completion.lua

```diff
@@ -154,10 +161,25 @@ function completion.shell(command, cur_pos, ncomp, shell)
         while true do
```

Comment:
I see.  I am more concerned about the `sort`, not the `unique`.
Could we use `uniq` instead?